pkgs/development/python-modules/caldav/default.nix

```diff
@@ -43,15 +44,9 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [
     pytestCheckHook
+    xandikos
```

Comment:
As is `xandikos` uses Python 3.11 even if we're testing `python310.pkgs.caldav`. Please fix!